### ISSUE TITLE
Nesting Filters, e.g. :cdata and :uglify-js

### ISSUE THREAD

**Issue by rudolfschmidt** [jadejs/jade#1997]
Hi,

you have the scenario to use jstransformers-uglify-js to uglify your readable js code, but also you want the uglified code to be wrapped in a cdata like

```jade
script
  :cdata:uglify-js
    (function() {
      console.log('test')
    })()
```

the result is 

```html
<script><![CDATA[]]>!function(){console.log("test")}();</script>
```

that is not valid cdata wrapped js code I guess.

Is there any solution or workaround?

**Comment by ForbesLindesay** [comment]
This is a bug and should be fixed when we release 2.0.0

**Comment by TimothyGu** [comment]
Now that jadejs/jade-parser#6 is merged, I'm closing this.

**Comment by TimothyGu** [comment]
Actually this doesn't work. Fix incoming.

**Comment by monteirocode** [comment]
+1

**Comment by TimothyGu** [comment]
Fixed in jadejs/jade-parser#9 and jadejs/jade-filters#1.
